test_pytest.py

```diff
@@ -108,3 +108,15 @@ def test_skill():
 
     response = app.test_client().get('/resume/skill')
     assert response.json[item_id] == example_skill
+
+def test_skill_delete():
```

Comment:
Same thing, does this test pass atm?

Reply:
Yes, the test does currently pass.

Reply:
In the pipeline it looks like this fails. The issue doesn't seem to be the content of the test, but the fact that there is now an id field in the skill model. Maybe you can add the id field in this test, as I think we have another issue that tracks what we should for the id addition

Reply:
Oops, my bad! I was looking at a different test